src/components/learnMore/components/styles.css

```diff
@@ -0,0 +1,46 @@
+.header {
+    background: url("./images/balaji-malliswamy-1281466-unsplash\ \(1\).jpg");
+    width: 100%;
+    height: 90vh;
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    background-attachment: fixed;
+}
+
+.body {
+    transition: background-color 1s ease;
+    background-color: white;
+    height: 100%;
+    display: flex; 
+    flex-wrap: wrap; 
+    justify-content: space-around;
+}
+.body:hover {
+    background-color: #e6e1e1;
+}
+
+.icon {
+    color: #106021;
+    height: 200px;
+    width: 350px;
+    padding-top: 50px;
+    font-size: 150px;
+}
+
+.body:hover .icon {
+    -webkit-transform: scale(1.6);
+    -ms-transform: scale(1.6);
+    transform: scale(1.6);
+}
+
+.text {
+    width: 900px;
+    height: 100%;
+    font-size: 90%;
+    text-align: center;
```

Comment:
You may try to` justify` the text

Reply:
@engreric sorry I don't understand

Reply:
Try setting the text align to  `justify`and see if it looks better